api/registry.py

```diff
@@ -334,12 +334,12 @@ def _send_registration_request(cls, register_url, payload, do_post):
 
         :return: Either a ProblemDetail or a requests-like Response object.
         """
-        # Allow 401 so we can provide a more useful error message.
+        # Allow 400 and 401 so we can provide a more useful error message.
```

Comment:
400 is returned for something like a missing help email. That error should be passed on to the admin interface.